cowprotocol/cowswap
# [Eth flow]#1290 disable for sc wallets

# Summary

Closes #1290 
Depends on https://github.com/cowprotocol/cowswap/pull/1367

Disabling Eth flow for Smart contract wallets

  # To Test

1. Enable the feature flag with `localStorage.setItem('enableEthFlow', '1')`
2. Sell ETH against another token
* Eth flow should be displayed
![Screen Shot 2022-11-10 at 14 55 36](https://user-images.githubusercontent.com/43217/201138672-63e9a64d-a12d-4168-9d50-5a003cbc3efa.png)
3. Connect an EOA
* Eth flow should still be available
4. Connect a smart contract wallet (Gnosis Safe for example)
* The previous wrap flow should be displayed
![Screen Shot 2022-11-10 at 14 55 50](https://user-images.githubusercontent.com/43217/201138997-369a1a4e-c5e3-4462-bd43-292dcc98acb5.png)
